beaker-vue/src/components/BeakerCodecellOutput.vue

```diff
@@ -5,10 +5,10 @@
             class="pi pi-spin pi-spinner busy-icon"
         />
         <div v-for="output of props.outputs" :key="output">
-            <div v-if="output.output_type == 'stream'" :class="output.name">{{ output.text }}</div>
-            <div v-else-if="output.output_type == 'display_data'" :class="output.name" v-html="renderResult(output)"></div>
-            <div v-else-if="output.output_type == 'execute_result'" :class="output.name" v-html="renderResult(output)"></div>
-            <div v-else-if="output.output_type == 'error'" :class="output.name" v-html="renderError(output)"></div>
+            <div v-if="output.output_type == 'stream'" :class="output.output_type">{{ output.text }}</div>
```

Comment:
Before, all these `:class` attributes that used `output.name` were empty output classes (class="").
I'm not sure if the intention was some other property, since I see classes below in the file such as `.stderr` and `.stdout`

@mattprintz Now thinking about it, I'm slightly confused on when the classes were being applied before, since I previously changed a color for `.stderr` and that worked. I have not seen the `stderr` getting attached recently on python errors (such as trying to access an undefined variable).

Maybe something changed recently, but looking to hear if you think this may disrupt other styles for other output types.